simpa_tests/automatic_tests/tissue_library/test_core_assumptions.py

```diff
@@ -6,7 +6,9 @@
 from simpa.utils import TISSUE_LIBRARY
 from simpa.utils.libraries.tissue_library import TissueLibrary
```

Comment:
remove from simpa.utils.libraries.tissue_library import TissueLibrary

Reply:
I think without further changes in the code we cannot just remove the import